data-otservbr-global/scripts/actions/other/teleport.lua

```diff
@@ -1,4 +1,6 @@
-local upFloorIds = {1948, 1968, 5542, 20474, 20475, 1968, 28656, 31262, 31129, 31130, 34243}
+local ladderTable = Game.getLadderTable()
```

Comment:
nit: "Table" is a leaky abstraction, `lua` happens to implement lists as tables, I think `Game.getLadderIds()` would be more abstract

Reply:
Ok thanks, I will modify.